.docs/content/1.installation/3.aws/0.aws-first-setup.md

```diff
@@ -0,0 +1,64 @@
+# AWS Setup 🚀
+
+This guide will help you install and configure the AWS CLI on your system and set up your AWS environment for use.
+
+## 1. Installation & Configuration
+
+Follow the official AWS CLI install guide [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+For Linux-based systems, you can use the following commands:
+
+```bash
+curl "https://awscli.amazonaws.com/aws-cli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+sudo ./aws/install
+```
+
+Ensure the AWS CLI is installed correctly by checking its version:
+
+```bash
+aws --version
+```
+
+## 2. Configure AWS CLI
+
+Once installed, you need to configure the AWS CLI with your credentials:
+
+```bash
+aws configure
+```
+
+During configuration:
+- Enter your **Access Key ID** and **Secret Access Key** (provided by AWS).
+- Choose a default region, e.g., *eu-west-3* (optional but recommended).
+- Specify the default output format (e.g., *json*, *table*, or *text*).
+
+To use AWS environnement, you can use the following:
```

Comment:
Before to be able to interact with AWS, a login using their SSO is needed. This is realized with the command: